web/src/js/components/Header/FlowMenu.tsx

```diff
@@ -108,6 +135,12 @@ const openInNewTab = (url) => {
 };
 
 function DownloadButton({ flow }: { flow: Flow }) {
+    const hasSingleFlowSelected = useAppSelector(
+        (state) => state.flows.selected.length === 1,
+    );
+
+    if (!hasSingleFlowSelected) return null;
```

Comment:
Is there a reason why this button is treated differently (removed and not disabled)?

Reply:
no my bad, thanks for spotting this :)